### DISCUSSION TITLE
Support for Manifest V3

### DISCUSSION THREAD

**Discussion by sum32139** [Authenticator-Extension/Authenticator#996] [Q&A]
Hi...
   Now that Google plans to [depreciate support for manifest V2](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/?hl=en), are there plans to upgrade ACC to V3?

**Comment by Sneezry** [comment]
Yes, we have the plan to migrate to V3. The previous block issue is the cached password in memory cannot be held in V3 since there is no backend agent. It seems the `chrome.storage.session` has already been available in the stable channel, we will start the work in the following days. Another remaining issue is the clear the cached password at a specific time since we have an auto-lock feature. Because no backend agent any longer, we need to figure out a way how to handle such a case.

**Reply by sum32139** [reply]
Thank you for quick response.  Any idea when you'll be able to upgrade to V3?

**Reply by sum32139** [reply]
Hi...I didn't see a response to my question so I thought I'd reach out again.  Both Edge & Chrome won't allow new extensions to be installed using a V2 manifest.  This is blocking our deployment of ACC.  Do you have a timeline on when you'll migrated to V3?

**Reply by mymindstorm** [reply]
Since this is an open source application we develop in our free time, there is not an estimated time to completion. Although, you may track the progress of MV3 via #1009.

**Comment by Sneezry** [comment]
MV3 released
